app/cli/cmd/root.go

```diff
@@ -269,12 +283,31 @@ func parseToken(token string) (*ParsedToken, error) {
 
 		switch aud[0].(string) {
 		case apiTokenAudience:
-			return &ParsedToken{Type: "api-token"}, nil
+			tokenID, ok := claims["jti"].(string)
```

Comment:
related to another of my comments, I think we should improve this method, it relies too much in casting, causing for example to return `nil` if the token doesn't have a property.

I think this could be improved by 

a - leveraging `claims.VerifyAudience(aud, true) bool`
b - once found, we could map to their actual claims so they are no longer a map of interface and can be accessed properly. 

I think this way the function will be less brittle, what do you think?